src/tools/configure.ts

```diff
@@ -13,7 +13,7 @@ function notEmpty(s: string) {
   return s.trim() !== "";
 }
 
-db.connect().then(async () => {
+runApp(async () => {
```

Comment:
Since the method is no longer passed on, I think it makes sense to wrap this in a `try catch`.